picard/tagger.py

```diff
@@ -480,6 +485,24 @@ def handle_command_fingerprint(self, argstring):
         for album_name in self.albums:
             self.analyze(self.albums[album_name].iterfiles())
 
+    def handle_command_from_file(self, argstring):
+        try:
+            lines = [line.strip() for line in open(argstring).readlines()]
+        except OSError:
+            log.error("Error opening command file: %s", argstring)
+            return
+        except Exception:
+            log.error("Error reading command file: %s", argstring)
+            return
+
+        for line in lines:
+            elements = shlex.split(line.strip())
+            if not elements:
+                continue
```

Comment:
In case you want to be able to have comment lines in the command file.  Since the first element should be a command (and none of them starts with an octothorpe), it should be safe to skip any line starting with an octothorpe (and/or other character if preferred) as a comment.

```suggestion
            # Skip blank lines or comment lines beginning with an octothorpe (#)
            if not elements or elements[0].startswith('#'):
                continue
```

Reply:
Thanks!